.prettierrc

```diff
@@ -1,4 +1,8 @@
 {
   "arrowParens": "avoid",
-  "singleQuote": true
+  "singleQuote": true,
+  "importOrder": [
+    "^[./]"
+  ],
+  "importOrderSeparation": true
 }
```

Comment:
In the client we have this configuration in `package.json`. We might want to do the same here to cut down on the proliferation of config files. In general we can do the same for other tools where a) the tool supports reading config from `package.json` and b) the config is small and obvious, so doesn't need comments.